CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 
 
+## Unreleased
+
+### Deprecated
+
+- The `debug_plugins` option for `PluginClient` is deprecated and will be removed in 2.0. Use the decorator design pattern instead like in https://github.com/php-http/HttplugBundle/blob/master/Collector/ProfilePlugin.php.
```

Comment:
when you are on a github page of the file, you can press the `y` key to get the link with the last commit id.